### PR TITLE
[doc] update get-starting.md

### PR DESCRIPTION
**Pull request by emerkfu**
# Proposed changes

Document formatting optimization, spelling correction.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/get-starting/get-starting.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ under the License.
 
 ## Stand-alone deployment
 
-**Before creating, please prepare the compiled FE/BE file, this tutorial will not repeat the compilation process.**
+**Before creating, please prepare the compiled FE / BE file, this tutorial will not repeat the compilation process.**
```
Comment (jackwener) [inline docs/en/docs/get-starting/get-starting.md:41]: It's redundant.

File: docs/zh-CN/docs/get-starting/get-starting.md [hunk 2]
```diff
@@ -32,16 +32,15 @@ under the License.
...
 3. 硬盘：100G（最低）400G（推荐）
-4. 平台：MacOS（Intel）、LinuxOS、Windows虚拟机
+4. 平台：MacOS（Intel）、LinuxOS、Windows 虚拟机
 5. 系统：CentOS（7.1及以上）、Ubuntu（16.04 及以上）
 6. 软件：JDK（1.8及以上）、GCC（4.8.2 及以上）
 
 ## 单机部署
 
-**在创建之前，请准备好已完成编译的FE/BE文件，此教程不再赘述编译过程。**
+**在创建之前，请准备好已完成编译的 FE / BE 文件，此教程不再赘述编译过程。**
```
Comment (jackwener) [inline docs/zh-CN/docs/get-starting/get-starting.md:41]: redundant

File: docs/zh-CN/docs/get-starting/get-starting.md [hunk 3]
```diff
@@ -32,16 +32,15 @@ under the License.
...
 
-
-**该教程为单节点、1FE 和 1BE 的混布教程，只需要一个节点，便于快速体验 Doris。**
+**该教程为单节点、1 FE 和 1 BE 的混合部署教程，只需要一个节点，便于快速体验 Doris。**
```
Comment (jackwener) [inline docs/zh-CN/docs/get-starting/get-starting.md:43]: don't remove the empty line.

**Review by jackwener — COMMENTED** [review]
Thanks contribution

File: docs/en/docs/get-starting/get-starting.md (hunk 1 above)
Comment (emerkfu) [reply docs/en/docs/get-starting/get-starting.md]: has been restored.

File: docs/zh-CN/docs/get-starting/get-starting.md (hunk 2 above)
Comment (emerkfu) [reply docs/zh-CN/docs/get-starting/get-starting.md]: has been restored.

File: docs/zh-CN/docs/get-starting/get-starting.md (hunk 3 above)
Comment (emerkfu) [reply docs/zh-CN/docs/get-starting/get-starting.md:44]: Deleted empty line have been added.

**Review by jackwener — APPROVED** [review]
LGTM
cc @hf200012

**Review by hf200012 — APPROVED** [review]
LGTM

**Comment by morningman** [comment]
@emerkfu please rebase to resolve conflicts.

**Pull request closed without merging**